component/src/androidTest/java/com/fioalpha/poc/component/ExampleInstrumentedTest.kt

```diff
@@ -1,13 +1,11 @@
 package com.fioalpha.poc.component
 
-import androidx.test.platform.app.InstrumentationRegistry
 import androidx.test.ext.junit.runners.AndroidJUnit4
-
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Assert.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fioalpha/CaseStudy/blob/6ce9cd99c0c1ee323fc1eba2aae62b43356b0111/component/src/androidTest/java/com/fioalpha/poc/component/ExampleInstrumentedTest.kt#L5"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>